content.js

```
@@ -64,6 +64,34 @@ async function loadDocs(url) {
   });
 }
 
+function sanitizeText(text) {
+  if (!text || typeof text !== 'string') return '';
+  
+  try {
+    // Remove or replace problematic characters
+    let sanitized = text
+      // Replace null bytes and other control characters
+      .replace(/[\x00-\x08\x0B\x0C\x0E-\x1F\x7F]/g, '')
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)